server/router/stats.ts

```diff
@@ -0,0 +1,155 @@
+import express, { Response } from 'express';
+import moment from 'moment-timezone';
+import { Condition } from 'dynamoose/dist/Condition';
+import { Stats } from '../models/stats';
+import { Ride } from '../models/ride';
+import * as db from './common';
+import { validateUser } from '../util';
+import { Driver, DriverType } from '../models/driver';
+
+const router = express.Router();
+const tableName = 'Stats';
+
+router.get('/:from/:to', validateUser('User'), async (req, res) => {
+  const { params: { from, to } } = req;
+
+  let date = from;
+  const dates = [];
+  while (date <= to) {
```

Comment:
`to` may not always be provided, which in that case `dates` should just be `[from]`